leo_filters/src/imu_filter.cpp

```diff
@@ -0,0 +1,267 @@
+// Copyright 2022-2023 Fictionlab sp. z o.o.
```

Comment:
Change the copyright year
```suggestion
// Copyright 2025 Fictionlab sp. z o.o.
```